bin/all_sky_search/pycbc_sngls_statmap

```diff
@@ -153,29 +146,66 @@ bg_far, fg_far = significance.get_far(
     fg_time,
     **significance_dict[ifo])
 
-# Cumulative array of exclusive background triggers and the number
-# of exclusive background triggers louder than each foreground trigger
-bg_far_exc, fg_far_exc = significance.get_far(
-    back_stat_exc,
-    fore_stat,
-    bkg_exc_dec_facs,
-    fg_time_exc,
-    **significance_dict[ifo])
-
 fg_far = significance.apply_far_limit(fg_far, significance_dict, combo=ifo)
 bg_far = significance.apply_far_limit(bg_far, significance_dict, combo=ifo)
-fg_far_exc = significance.apply_far_limit(fg_far_exc, significance_dict, combo=ifo)
-bg_far_exc = significance.apply_far_limit(bg_far_exc, significance_dict, combo=ifo)
 
 bg_ifar = 1. / bg_far
 fg_ifar = 1. / fg_far
-bg_ifar_exc = 1. / bg_far_exc
-fg_ifar_exc = 1. / fg_far_exc
 
 f['background/ifar'] = conv.sec_to_year(bg_ifar)
-f['background_exc/ifar'] = conv.sec_to_year(bg_ifar_exc)
+
 f.attrs['background_time'] = fg_time
 f.attrs['foreground_time'] = fg_time
+
+# Find foreground triggers with IFAR > 1 per livetime and remove from
+# the exclusive background (not equal, as that could give an infinite
+# loop if using count_n_louder)
+fg_ifar_exc = copy.deepcopy(fg_ifar)
```

Comment:
Why is this copying done here but other copying was done earlier? 
Would be easier to understand the whole thing if all the 'exc' copying is done in one place, IMO.  Also according to the principle of don't create a variable yet if the first place you use it is 40 lines later (back_stat_exc etc).

Reply:
Added now